src/bundle/Resources/views/themes/admin/ui/component/dropdown/dropdown.html.twig

```diff
@@ -24,19 +25,21 @@
     {{ translation_domain is same as(false) ? label : label|trans({}, translation_domain) }}
 {% endmacro %}
 
-<div
-    class="ibexa-dropdown
-        {{ multiple ? "ibexa-dropdown--multi" : "ibexa-dropdown--single" }}
-        {{ custom_init|default(false) ? "ibexa-dropdown--custom-init" }}
-        {{ is_dynamic ? "ibexa-dropdown--dynamic" }}
-        {{ is_small|default(false) ? "ibexa-dropdown--small" }}
-        {{ is_ghost|default(false) ? "ibexa-dropdown--ghost" }}
-        {{ is_selector ? "ibexa-dropdown--selector" }}
-        {{ no_items and not is_dynamic or is_disabled ? "ibexa-dropdown--disabled" }}
-        {{ class|default('') }}
-    "
-    {% if is_hidden|default(false) %}hidden="hidden"{% endif %}
->
+{% set attr_dropdown = attr_dropdown|default({})|merge({
```

Comment:
```suggestion
{% set attr = attr|default({})|merge({
```

Reply:
attr is already used in this scope for input